app/assets/stylesheets/application.scss

```diff
@@ -27,6 +27,19 @@
 a:hover {
   text-decoration: underline;
 }
+
+footer, nav {
+  span.spacer{
+    margin: 0em 0.5em 0em 0.5em;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `0em 0.5em` instead of `0em 0.5em 0em 0.5em`<br>`0em` should be written without units as `0`